radis/test/misc/test_utils.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+import pytest
+import unittest
+from os.path import dirname, basename, join
+from os import listdir
+
+# Test to compare basenames of files whose names match the regular expression passed.
+def test_files_from_regex(*args, **kwargs):
+    from radis.misc.utils import get_files_from_regex
+    from radis.test.utils import getTestFile
+    from os.path import dirname, basename
+    import fnmatch
+
+    # path1 is for finding all the files with .par extension having regular expression *.par
+    path1 = getTestFile("*.par")
+
+    path_test = get_files_from_regex(path1)
+
+    # This list will have to be updated if more .par files are added/removed in/from the folder
+    path_actual = [
+        "hitran_co2_626_bandhead_4165_4200nm.par",
+        "hitran_2016_H2O_2iso_2000_2100cm.par",
+        "hitran_CO_fragment.par",
+        "hitran_co_3iso_2000_2300cm.par",
+        "hitran_CO2_fragment.par",
+    ]
+    path_actual.sort()
+
+    for i in range(
+        len(path_test)
+    ):  # finding basename because the function `get_file_from_regex` returns the absolute path
+        path_test[i] = basename(path_test[i])
+
+    path_test.sort()
+    assert all([x == y for x, y in zip(path_test, path_actual)])
```

Comment:
Although it's nice to have a single-line code, having a single assert for all test cases can make it harder to debug if the assertion test fails.  Can you do that in the loop above?

Reply:
I have another assertion below this one, for the second test case.

Reply:
Understood and done!